.build/build-resolver.xml

```diff
@@ -246,6 +247,14 @@
             <file file="${local.repository}/org/apache/cassandra/deps/sigar-bin/libsigar-x86-linux.so"/>
             <file file="${local.repository}/org/apache/cassandra/deps/sigar-bin/libsigar-x86-solaris.so"/>
         </copy>
+
+        <copy todir="${build.lib}/corretto" quiet="true">
+            <file file="${local.repository}/software/amazon/cryptools/AmazonCorrettoCryptoProvider/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-x86_64.jar"/>
+            <file file="${local.repository}/software/amazon/cryptools/AmazonCorrettoCryptoProvider/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-aarch_64.jar"/>
+        </copy>
+
+        <delete file="${build.lib}/AmazonCorrettoCryptoProvider-2.2.0-linux-x86_64.jar" failonerror="false"/>
+        <delete file="${build.lib}/AmazonCorrettoCryptoProvider-2.2.0-linux-aarch_64.jar" failonerror="false"/>
```

Comment:
why the two deletes?

Reply:
because when doing `ant jar`, since that dependency is in pom.xml, resolver will copy all libraries as dependencies to `libs`. But we are also copying x86 and aarch to `corretto` dir. So you would have there these files two times.

Reply:
cool.  add a comment please :)

Reply:
Perhaps it would be better to just move these jars to the lib/{$platform} folder after they have been resolved and downloaded into the `jar` folder, no?